### PR TITLE
Include More Azure CLI tests

### PR DESCRIPTION
**Pull request by yarunachalam**
RBAC test and load balancer test

- Verification run:(https://openqa.suse.de/tests/11244444#settings - RBAC)
https://openqa.suse.de/tests/11246854#settings - Load Balancer

### CODE REVIEW

File: data/publiccloud/azure_lb.sh [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+iplist=$(az network lb rule list -g ${grpname} --lb-name ${loadBalancer} --query "[].{name:name}" -o tsv | head -n 1)
+cmd_status "az_net_lb_rule_show" az network lb rule show \
+   --name "${iplist}" \
+   --resource-group "${grpname}" \
+   --lb-name "${loadBalancer}" \
+   --output table
+
+# Create three NAT rules for port 22.
+echo "Creating three NAT rules named ${loadBalancerRuleSSH}1 - ${loadBalancerRuleSSH}3"
+for i in `seq 1 3`; do
```
Comment (ricardobranco777) [inline data/publiccloud/azure_lb.sh:137]:
```suggestion
for i in {1..3}; do
```
No need to call an external command.  I think `for i in 1 2 3` is even more readable.

File: data/publiccloud/azure_lb.sh [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+   --source-address-prefix '*' \
+   --source-port-range '*' \
+   --destination-address-prefix '*' \
+   --destination-port-range 80 \
+   --access allow --priority 2000 \
+   --output table
+
+# Create three virtual network cards and associate with public IP address and NSG.
+echo "Creating three NICs named ${nic}1 - ${nic}3 for $vNet and $subnet"
+for i in `seq 1 3`; do
```
Comment (ricardobranco777) [inline data/publiccloud/azure_lb.sh:193]: Ditto above.

File: data/publiccloud/azure_lb.sh [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+echo "LB availability-set and list-sizes list & show"
+echo "**********************************************"
+for iplist in $(az vm availability-set list -g ${grpname} --query "[].{name:name}" -o tsv); do
+    cmd_status "az_vm_avset_show" az vm availability-set show --name ${iplist} -g ${grpname} -o table
+    cmd_status "az_vm_avset_list_sizes" az vm availability-set list-sizes --name ${iplist} -g ${grpname}
+done
+
+# Create three virtual machines, this creates SSH keys if not present.
+echo "Creating three VMs named ${vmname}1 - ${vmname}3 with ${nic}1 - ${nic}3 using ${vmximagename}"
+for i in `seq 1 3`; do
```
Comment (ricardobranco777) [inline data/publiccloud/azure_lb.sh:224]: Ditto above.

**Review by ricardobranco777 — COMMENTED** [review]
Please provide a VR. Added a suggestion. Please run the `shellcheck` utility against the scripts and fix the warnings. There are some like `loadBalancerRuleWeb is referenced but not assigned`

**Comment by yarunachalam** [comment]
> loadBalancerRuleWeb

I can run the shellcheck. but
assigned and referenced 
 `loadBalancerRuleWeb="msdocs-load-balancer-rule-port80-${randomIdentifier}"`
 
 
 `cmd_status "az_net_lb_rule_create" az network lb rule create \
   --resource-group "${grpname}"  \
   --lb-name "${loadBalancer}" \
   --name "${loadBalancerRuleWeb}" \
   --protocol tcp \
   --frontend-port 80 \
   --backend-port 80 \
   --frontend-ip-name "${frontEndIp}" \
   --backend-pool-name "${backEndPool}" \
   --probe-name "${probe80}" \
   --output table
`

**Comment by pdostal** [comment]
> I can run the shellcheck. but
> assigned and referenced

Sorry, I don't understand.

**Comment by pdostal** [comment]
I know that this is work in progress but before merging the pull request title and pull request description must be updated as well as commit title adjusted.

**Comment by yarunachalam** [comment]
> Please provide a VR. Added a suggestion. Please run the `shellcheck` utility against the scripts and fix the warnings. There are some like `loadBalancerRuleWeb is referenced but not assigned`

shellcheck completed for both azure_lb.sh and azure_rbac.sh

File: data/publiccloud/azure_lb.sh (hunk 1 above)
Comment (yarunachalam) [reply data/publiccloud/azure_lb.sh]: done

File: data/publiccloud/azure_lb.sh (hunk 3 above)
Comment (yarunachalam) [reply data/publiccloud/azure_lb.sh]: done

**Pull request merged**